src/components/CreateGoalForm/RadioGroup.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from 'react';
+import RadioGroup from 'react-native-radio-buttons-group';
+import { Text, View, StyleSheet } from 'react-native';
+
+const radioButtonsData = [
+  {
+    id: '1', // acts as primary key, should be unique and non-empty string
+    label: 'Existing',
+    value: 'existing',
+  },
+  {
+    id: '2',
+    label: 'Custom',
+    value: 'custom',
+  },
+];
+
+export default function App() {
```

Comment:
```suggestion
export default function App() {

why it's named as App? 
```

Reply:
My bad, missed to change this while switching from snack expo to local.